plutus-pab/src/Cardano/Node/Server.hs

```diff
@@ -75,7 +75,7 @@ main trace MockServerConfig { mscBaseUrl
     serverHandler <- liftIO $ Server.runServerNode trace mscSocketPath mscKeptBlocks (_chainState appState)
     serverState   <- liftIO $ newMVar appState
     handleDelayEffect $ delayThread (2 :: Second)
-    clientHandler <- liftIO $ Client.runClientNode mscSocketPath mscSlotConfig (updateChainState serverState)
+    clientHandler <- liftIO $ Client.runClientNode mscSocketPath mscSlotConfig (\_ _ -> pure ())
```

Comment:
What's the `clientHandler` used for - just for submitting blocks?

Reply:
Yes.

Reply:
Lets merge it like this for now (after we validate the fix), then I will think about splitting it into two parts, trying to avoid `\_ _ -> pure()` which happens quite a bit.

Reply:
Yeah we probably need two clients, one for each protocol (tx submission and chain sync)